core/shared/src/main/scala/me/shadaj/scalapy/interpreter/CPythonInterpreter.scala

```diff
@@ -318,14 +353,29 @@ object CPythonInterpreter {
 
         CPythonAPI.PyErr_Fetch(pType, pValue, pTraceback)
 
+        val tracebackStrOpt = Traceback.tryUse { tracebackOpt =>
+          for {
+            traceback <- tracebackOpt
+            pyTraceback <- pointerToPointerToPyValueOpt(pTraceback)
+          } yield {
+            "Traceback (most recent call last):" + System.lineSeparator() +
+              traceback.format_tb(pyTraceback)
+                .__scalapy_value
+                .getSeq[PyValue](PyValue.fromBorrowed(_), null)
```

Comment:
Not sure about the `fromBorrowed` here, I didn't think much about it… (yet)

Reply:
`fromBorrowed` should be fine, the only reason this lambda parameter exists is to enable fast paths that avoid allocating `PyValue`s when reading Python collections into Scala copies.